openmetadata-ui/src/main/resources/ui/src/components/PipelineDetails/PipelineDetails.component.tsx

```diff
@@ -569,17 +584,18 @@ const PipelineDetails = ({
         key: t('label.name'),
         dataIndex: 'name',
         title: t('label.name'),
-        render: (name, record) => (
-          <Link target="_blank" to={{ pathname: record.taskUrl }}>
+        render: (name, record) =>
+          isEmpty(record.taskUrl) || isUndefined(pipelineHostPort) ? (
             <span>{name}</span>
-            <SVGIcons
-              alt="external-link"
-              className="align-middle m-l-xs"
-              icon="external-link"
-              width="16px"
-            />
-          </Link>
-        ),
+          ) : (
+            <Link
+              className="flex items-center gap-2"
+              target="_blank"
+              to={{ pathname: pipelineHostPort + record.taskUrl }}>
```

Comment:
I think we decide to not to have any logic for link creation?

Reply:
Right. Thanks for catching this. I missed changing this later discussion. Will do the changes.